tests/integration/test_system_containers_images.sh

```diff
@@ -105,31 +105,6 @@ ${ATOMIC} pull --storage ostree docker:atomic-test-system:latest
 ${ATOMIC} images list -f type=ostree > ${WORK_DIR}/images.out
 assert_matches "atomic-test-system" ${WORK_DIR}/images.out
 
-
-# 4.1 Check that the virtual size of the imported image is the same as showed for Docker
-${ATOMIC_NO_DEBUG} images list --json > ${WORK_DIR}/images.json
-
-cat > test_images_size.py <<EOF
-#!/usr/bin/env python
-import json
-import sys
-def make_number(x):
-    foo = x.replace("MB", "").replace("KB", "").replace(" ","")
-    return float(foo)
-
-def is_different(x, y):
```

Comment:
how much is different when copying with Skopeo?  If it helps we can adjust the threshold

Reply:
Hmm, I am not entirely certain yet. I have only investigated a bit for images like `hello-world` or `docker-fedora`, and the difference seems to be varying. For hello-world, it is 2KB extra, and for docker-fedora, it was like 15 Mb extra. I would assume if we want to check for threshold, we might need to calculate a certain percentage difference. 

Let me investigate that a bit.

Reply:
yea.. sorry, investigated that a bit, seems like there wasn't an obvious relationship between image size. I have pulled 4 different images from docker to ostree. Here are the results of those:
```
[root@localhost atomic]# atomic images list | grep atomic-test-4
   atomic-test-4               latest   39f9d7398cc3   2018-02-21 22:47   207.2 MB       docker    
   atomic-test-4               latest   99cab1cd1273   2018-02-28 19:59   215.75 MB      ostree

[root@localhost atomic]# atomic images list | grep docker.io/hello-world
   docker.io/hello-world       latest   f2a91732366c   2017-11-21 00:23   1.85 KB        docker    
   docker.io/hello-world       latest   8072a54ebb3b   2018-02-28 19:48   985.0 B        ostree

[root@localhost atomic]# atomic images list | grep atomic-test-system
   atomic-test-system          latest   bb3c8d007810   2018-02-28 19:53   228.09 MB      docker    
   atomic-test-system          latest   56db3c2331ac   2018-02-28 19:54   236.74 MB      ostree

[root@localhost atomic]# atomic images list | grep atomic-test-secret
   atomic-test-secret          latest   6a5b9db82a6f   2018-02-28 19:53   68.0 B         docker    
   atomic-test-secret          latest   1b6853fe7ec4   2018-02-28 19:47   68.0 B         ostree 
```

One thing I do notice tho, we are copying blob of sha256 checksum, and config each time via skopeo, may be we have a different way of calculate it between skopeo and docker?

Reply:
yea, I believe so.. cuz we are only testing it against one image right? For now, to keep the code there, we can hard code the threshold, and I am going to have it to be 4%( that is the difference for atomic-test-system). Will push a fixup soon :)

Reply:
Force pushed a new version below with size threshold relaxed. PTAL :)